sdk/boards/ibex-arty-a7-100.json

```diff
@@ -23,6 +23,14 @@
         "gpio_led0" : {
             "start": 0x8f00f000,
             "length": 0x800
+        },
+        "dmb_cfg": {
+            "start": 0x8f0f0000,
+            "length": 0x1000
+        },
+        "dmb": {
+            "start": 0x90000000,
+            "length": 0x100000
```

Comment:
This hunk looks unrelated; am I missing something?

Reply:
Yup, that was accidentally added.